### DISCUSSION TITLE
LG AC can't be controlled

### DISCUSSION THREAD

**Discussion by maxxom64** [nVuln/homebridge-lg-thinq#117] [Q&A]
I have installed HOOBS and your Plugin. When I configure my LG Account, it will find all my 4 AC devices. And will create Accessoires as well. It will show a Switch and a Box per Accessoire. But the Switch will not turn on the AC. The Accessoires are also not displayed in the Accessoires section in the configuration.

The Log shows the following:
3.1.2022, 11:03:49 [LGThinQ] Loading accessory from Homebridge cache: Wohnen
3.1.2022, 11:03:52 [LGThinQ] Successfully connected to the ThinQ API.
3.1.2022, 11:03:52 [LGThinQ] [Wohnen] Setting up device!
3.1.2022, 11:03:52 [LGThinQ] Restoring existing accessory: xxxx: Wohnen (AC RAC_056905_WW)
3.1.2022, 11:03:52 [LGThinQ] Start MQTT listener for thinq2 device
3.1.2022, 11:03:53 [LGThinQ] Successfully connected to the MQTT server.

which looks correct for me.

What is the problem here?

**Comment by nVuln** [comment]
any error log?
